.github/workflows/cpplint.yml

```diff
@@ -0,0 +1,21 @@
+name: cpplint
```

Comment:
if you activate Github Actions on this repo (free), will run a cpplint for each push and PR.